packages/roosterjs-plugin-picker/lib/PickerPlugin.ts

```diff
@@ -1,3 +1,4 @@
+import isCharacterValue from 'roosterjs-editor-core/lib/eventApi/isCharacterValue';

```

Comment:
Can these be exported from the root? Does it make sense to export these from the root?

Reply:
Agree

Reply:
Sounds good